msb/camera/msb_camera.py

```diff
@@ -1,78 +1,109 @@
+import cv2
 import datetime
 import json
 import os
-import picamera
+from picamera2 import MappedArray, Picamera2
+from picamera2.encoders import H264Encoder
 import signal
 import sys
 import time
 
-timeformat = "%Y%m%dT%H%M%S%z"
-split_interval = 60 # seconds
+from msb.camera.config import CameraConf
+from msb.config import load_config
+
+timeformat = "%Y-%m-%d %X"
+color = (0, 255, 0)
+origin = (0, 30)
+font = cv2.FONT_HERSHEY_SIMPLEX
+scale = 1
+thickness = 2
```

Comment:
These are defined here as well as in the config, If possible remove these and use the config values.
Also font in the config is a dict, here it is the name of the font.

Reply:
renamed `font` to `font_name`. I've added these as explicit global variables due to the call back function not being able to process any arguments.